jupyterlite_sphinx/jupyterlite_sphinx.py

```diff
@@ -570,7 +571,7 @@ def run(self):
         # Button with the onclick event to swap examples with embedded notebook.
         try_it_button_html = (
             '<div class="try_examples_button_container">'
-            '<button class="try_examples_button" '
+            '<button class="try_examples_button hidden" '
```

Comment:
```suggestion
            '<button class="try_examples_button hidden transparent" '
```

(see other suggestions above)